.docs/conf.py

```diff
@@ -41,6 +41,14 @@
 # This pattern also affects html_static_path and html_extra_path.
 exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
 
+autodoc_default_options = {
+    'member-order': 'bysource',
+    'undoc-members': None,
```

Comment:
@daremiyo This is a total nitpick, but I'm wondering if you would be able to clarify why you need to specify this `undoc-members` option if it's just being set to `None`? If the intent is to exclude functions without a docstring, the [Sphinx docs](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html) seem to indicate you could do `no-undoc-members`

Reply:
@helrond I used undoc-members because it was one of the supported options for the autodocs default options - https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_default_options . all members in the index file doesn't bring in members inherited from base classes, so the undoc-members is generating documentation for a lot of the code in the serializers file that is under the serializer class

Reply:
Ah thanks for the clarification and pointing me to that spot in the docs!